### PR TITLE
Delim fixes, Sorting, Mutations to WS GraphQL

### PR DESCRIPTION
**Pull request by dwsutherland**
This PR is an accompaniment to changes within the pending `cylc-flow` [Incremental data-store generation](https://github.com/cylc/cylc-flow/pull/3202) PR, namely:
- Data element ID delimiter.
- Sorting.

This involved updating the resolvers, so I took the opportunity to:
- Send mutation requests directly to the GraphQL endpoint of the relevant workflow services.
(avoiding use of the delimiter altogether)

Ran tests in GraphiQL, everything working as far as I can tell!

### CODE REVIEW

**Comment by dwsutherland** [comment]
Travis will fail until https://github.com/cylc/cylc-flow/pull/3202 is in.

**Comment by oliver-sanders** [comment]
Have had a play with this, I'm getting this error message from the Cylc-UI.

`Network error: Response not successful: Received status code 400`

**Comment by kinow** [comment]
Re-opening PR to trigger Travis build. It should pass with no issues now :+1:

**Comment by dwsutherland** [comment]
> Re-opening PR to trigger Travis build. It should pass with no issues now 👍

Needs https://github.com/cylc/cylc-flow/pull/3202 to go in first.

**Comment by kinow** [comment]
Just realized it was not due that `six` & `graphql-relay`, but that it's actually depending on the resolvers and other changes in 3202. Sorry @dwsutherland !

File: cylc/uiserver/resolvers.py [hunk 1]
```diff
@@ -17,7 +17,10 @@
 """GraphQL resolvers for use in data accessing and mutation of workflows."""
 
 from fnmatch import fnmatchcase
```
Comment (kinow) [inline cylc/uiserver/resolvers.py]: You removed some code block in this change that made this import irrelevant :+1: 

So ` from fnmatch import fnmatchcase` can be removed now.

File: cylc/uiserver/resolvers.py [hunk 2]
```diff
@@ -87,60 +98,52 @@ def __init__(self, ws_mgr, data_mgr):
...
+    async def mutator(self, info, command, w_args, args):
         """Mutate workflow."""
-        w_ids = [flow.workflow.id
-                 for flow in await self.get_workflow_msgs(w_args)]
-        return self.ws_mgr.multi_request(command, w_ids, args)
+        w_ids = [
+            flow.workflow.id
+            for flow in await self.get_workflow_msgs(w_args)]
+        # Pass the request to the workflow GraphQL endpoints
+        req_str, vars, _, _ = info.context.get('handler').graphql_params
```
Comment (matthewrmshin) [inline cylc/uiserver/resolvers.py]: `vars` is a Python built-in. Should avoid using as name of local variable.

**Pull request merged**